docs/UserGuide.md

```diff
@@ -357,32 +379,30 @@ When you enter the above command, all dates and the Facebook handle associated w
 
 <br>
 
-### Delete a person : `delete INDEX` <a name="delete"></a>
+### Delete a contact : `delete INDEX` <a name="delete"></a>
 
-Deletes the specified person from SociaLite.
+This command deletes the specified contact from SociaLite.
 
 <div markdown="block" class="alert alert-primary">
 **:mag_right: Format:**
 `delete INDEX`
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown on the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* This command deletes the contact at the specified `INDEX`.
+* The index **must be a positive integer** 1, 2, 3, ...
 </div>
 
 <div markdown="block" class="alert alert-success">
 **:heavy_check_mark: Example:** `list` followed by `delete 7` 
 
-Deletes the 7th person on the displayed person list.
+Deletes the 7th contact on the displayed contact list.
```

Comment:
Maybe we can change this to "This command will delete the 7th contact on the displayed contact list."